### PR TITLE
Create vip template

### PR DESCRIPTION
**Pull request by DavidKnott**
### - What I did
Create VIP (Viper Improvement Proposal) template to facilitate discussion around the addition features to Viper. 

### - How I did it
I looked at the Ethereum EIP (Ethereum Improvement Proposal) template and took what I felt would be relevant for Viper
### - How to verify it
Read the template
### - Description for the changelog
None
### - Cute Animal Picture
image

### CODE REVIEW

File: .github/VIP_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+    Requires (*optional): <VIP number(s)>
+    Replaces (*optional): <VIP number(s)>
+
+
+## Simple Summary
+"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the VIP.
+
+## Abstract
+A short (description of the technical issue being addressed.
```
Comment (chgue) [inline .github/VIP_TEMPLATE.md]: Small typo: "("

File: .github/VIP_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Motivation
+The motivation is critical for VIPs that add or change Viper's functionality. It should clearly explain why the existing Viper functionality is inadequate to address the problem that the VIP solves. as well as how the VIP is in line with Viper's goals and design philosopy.
```
Comment (chgue) [inline .github/VIP_TEMPLATE.md]: Small typo: "solves. as well"

**Comment by DavidKnott** [comment]
@chgue Made the changes you requested and added a link to it in  `contributing.rst`
Thanks for the feedback!

File: docs/contributing.rst [hunk 3]
```diff
@@ -17,10 +17,18 @@ In particular, we need help in the following areas:
 * Responding to questions from other users on `StackExchange
   <https://ethereum.stackexchange.com>`_ and the `Viper Gitter
   <https://gitter.im/ethereum/viper>`_
+* Suggesting Improvements
+<https://github.com/ethereum/viper/tree/master/.github/VIP_TEMPLATE.md`_
```
Comment (chgue) [inline docs/contributing.rst]: @DavidKnott Thanks for adding it to the docs! There is a closing ">" missing before the "`" here and in the change below.

**Comment by DavidKnott** [comment]
@chgue Great catch, I added the `>`, merging it in.

**Pull request merged**